database/index.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+const data = require('../sampleData.js');
+
+const connection = mongoose.connect('mongodb://localhost/musicPlayer', { useMongoClient: true });
+
+console.log('GRRRREEAAAAT SUCCESSSSS!!!!!');
+
+const songSchema = mongoose.Schema({
+  // TODO: your schema here!
+  id: {
+    type: Number, unique: true, required: true, dropDups: true
+  },
+  title: String,
+  artist: String,
+  album: String,
+  genre: String
+});
+
+const Song = mongoose.model('Song', songSchema);
+
+const save = (err, inputSong) => {
+  if (err) {
+    console.log(err);
+  } else {
+    const song = new Song(inputSong);
+    song.save(() => {
```

Comment:
I think it'd be best to use node error-first handling in your save function (two cases: error saving and successfully saved)

Reply:
I initially had that two step error handling, eslint was complaining about err being already defined in an earlier scope in this line   song.save((err) => {  and so I deleted that part. As a result of that, I figure it would never get to the 2nd part if there was an error to begin with and erased the 2nd error handler in that callback. I guess I can re-write it in and name the 2nd handler error instead of err.